### DISCUSSION TITLE
Custom Race?

### DISCUSSION THREAD

**Discussion by barraIhsan** [monkeytypegame/monkeytype#1071] [Ideas] [answered]
So there we can make our own race (with friends), we can select the time, the fundbox, and the language.

**Comment by frogfrogfrogfrogfrog** [comment, marked as answer]
So your talking about multiplayer right? Currently it's still in the works and has been for quite some time so I don't think you'll be able to have this feature anytime soon. 

Although you can somewhat check the progress through the project's page and the one that's named "Tribe" is the multiplayer project. It seems that @Miodec is nearly finished but perhaps there's many bugs or unfinished features.

**Reply by mxstbr** [reply]
What's the latest update on multiplayer Monkeytype? I'd love to use it with my friends and coworkers! @Miodec

**Reply by barraIhsan** [reply]
yes, I would see an update for tribe!

**Reply by frogfrogfrogfrogfrog** [reply]
Tribe has a functional version out right now actually. It's not 100% fully finished from what I know but you can play with your friends at https://dev.monkeytype.com/ although tribe hasn't been imported into the main website.
